apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/constants.ts

```diff
@@ -8,6 +8,16 @@ export const IMAGE_UPLOAD_ACCEPTED_MIME_TYPES = [
   "image/bmp",
   "image/webp",
 ]
```

Comment:
nit: (outside of the diff) I think `IMAGE_UPLOAD_ACCEPTED_MIME_TYPES` is no longer used?

Reply:
good catch, deleted!